aten/src/ATen/native/native_functions.yaml

```diff
@@ -3375,6 +3375,30 @@
     CPU, CUDA: sub_
     SparseCPU, SparseCUDA: sub_sparse_
 
+- func: heaviside.Tensor_out(Tensor self, Tensor val, *, Tensor(a!) out) -> Tensor(a!)
+  dispatch:
+    CPU, CUDA: heaviside_tensor_out
+
+- func: heaviside.Scalar_out(Tensor self, Scalar val, *, Tensor(a!) out) -> Tensor(a!)
+  dispatch:
+    CPU, CUDA: heaviside_scalar_out
+
+- func: heaviside.Tensor(Tensor self, Tensor val) -> Tensor
+  use_c10_dispatcher: full
+  variants: function, method
+  device_guard: False
```

Comment:
I wouldn't bother trying to manually manipulate device guards here or in the following function.

Reply:
Addressed!